framework/src/application/network/network.ts

```diff
@@ -90,89 +91,104 @@ interface P2PRequest {
 }
 
 export class Network {
-	private readonly options: NetworkConfig;
-	private readonly channel: InMemoryChannel;
-	private readonly logger: Logger;
-	private readonly storage: Storage;
-	private secret: number | null;
-	private p2p!: liskP2p.P2P;
+	private readonly _options: NetworkConfig;
+	private readonly _channel: InMemoryChannel;
+	private readonly _logger: Logger;
+	private readonly _networkDB: KVStore;
+	private _secret: number | null;
+	private _p2p!: liskP2P.P2P;
 
 	public constructor({
 		options,
 		channel,
 		logger,
-		storage,
+		networkDB,
 	}: NetworkConstructor) {
-		this.options = options;
-		this.channel = channel;
-		this.logger = logger;
-		this.storage = storage;
-		this.secret = null;
+		this._options = options;
+		this._channel = channel;
+		this._logger = logger;
+		this._networkDB = networkDB;
+		this._secret = null;
 	}
 
 	public async bootstrap(): Promise<void> {
-		// Load peers from the database that were tried or connected the last time node was running
-		const previousPeersStr = await this.storage.entities.NetworkInfo.getKey(
-			NETWORK_INFO_KEY_TRIED_PEERS,
-		);
-		let previousPeers: ReadonlyArray<liskP2p.p2pTypes.ProtocolPeerInfo> = [];
+		let previousPeers: ReadonlyArray<liskP2P.p2pTypes.ProtocolPeerInfo> = [];
 		try {
+			// Load peers from the database that were tried or connected the last time node was running
+			const previousPeersStr = await this._networkDB.get<string>(
+				NETWORK_INFO_KEY_TRIED_PEERS,
+			);
 			// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
 			previousPeers = previousPeersStr ? JSON.parse(previousPeersStr) : [];
-		} catch (err) {
-			this.logger.error(
-				{ err: err as Error },
-				'Failed to parse JSON of previous peers.',
-			);
+		} catch (error) {
+			if (!(error instanceof NotFoundError)) {
+				this._logger.error(
```

Comment:
I think error is opposite. `if (!(error instanceof NotFoundError))` then it's parsing error or other database error.
In that case i think it make sense to log, but `else` case is not needed as it's expected error